Bottom: 9afe0ea18a95d1d48f1b9ae8c0bf9626f224e875
Top:    6ad097cd18cce6245a23fcd6d1a92cbbcbc45b88
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-21 11:49:50 -0500

Update ptxdistrc filename


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
similarity index 100%
rename from docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
rename to docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
